cedar-policy-validator/src/extensions/ipaddr.rs

```diff
@@ -21,9 +21,13 @@ use cedar_policy_core::evaluator::RestrictedEvaluator;
 use cedar_policy_core::extensions::{ipaddr, Extensions};
 use std::str::FromStr;
 
+/// Note on panic safety
+/// This module depends on the cedar parser only constructings ASTs with valid extension calls
 /// If any of the panics in this file are triggered, that means that this file has become
 /// out-of-date with the ipaddr extension definition in CedarCore.
```

Comment:
as-is, the `///` attaches this as a doc comment to `get_argument_types()`.  Maybe that's fine, in which case we don't need the extra backreference in the docs on `get_argument_types()`.  Or, we could use `//!` here to make a doc comment on the entire module; or, we could use `//` to make this block not a doc comment at all

Reply:
I think module comment is the way to go, and is what I assumed it was doing when I edited to match the style of existing comment.